samples/Gaussian_initial_state/read_state.py

```diff
@@ -0,0 +1,24 @@
+import matplotlib.pyplot as plt
+
+# read the probability amplitudes of result state into list
+list = []
+with open('wavefcn_recursive.txt', 'r', encoding="utf8") as file:
```

Comment:
As suggested above, I would name this `"gaussian_wavefcn.txt"` instead

Reply:
Done!